lib/client-manager.js

```diff
@@ -15,7 +15,8 @@ function ClientManager (opts) {
     // only make requests to external sites every 15 seconds.
     ttl: 15,
     // how long should we cache if request to upstream server failed?
-    ttlFailure: 2
+    ttlFailure: 2,
+    packageDoc: require('./package-doc')()
```

Comment:
I've been enjoying using `process.env` for dependency injection, good way not to break the rule of Law of Demeter anywhere -- for tests though, I like your idea that we'd kick `opts` along. will patch.